js/src/secureApi.js

```diff
@@ -18,9 +18,9 @@ import { uniq } from 'lodash';
 
 import Api from './api';
 import { LOG_KEYS, getLogger } from '~/config';
+import store from 'store';
```

Comment:
Generally prefer package imports to go at the top, then local src, then local to component.